dmsite/venv/src/Authentication/LoginTextField.js

```diff
@@ -0,0 +1,67 @@
+/**
+ * Created by christinadaley on 1/18/19.
+ */
+
+import React from "react";
+import { render, ReactDOM } from "react-dom";
+import { TransitionMotion, spring } from 'react-motion';
+import { Component } from 'reactstrap';
+import { withRouter } from "react-router-dom";
+import {Checkbox, Form, Icon, Input, Button} from 'antd';
+
+class TextField extends React.Component {
+    handleSubmit = (e) => {
+        e.preventDefault();
+        this.props.form.validateFields((err, values) => {
+            if (!err) {
+                console.log('Received values of form: ', values);
+            }
+        });
+    }
+
+    render() {
+        const { getFieldDecorator } = this.props.form;
+        return (
+            <Form onSubmit={this.handleSubmit} className="login-form">
+                <Form.Item>
+                    {getFieldDecorator('email', {
+                        rules: [{ required: true, message: 'Please input your email!' }],
+                    })(
+                        <Input prefix={<Icon type="user" style={{ color: 'rgba(0,0,0,.25)' }} />} placeholder="Email" />
```

Comment:
This was echoed in a previous pull request, but it's a tad annoying to use both single and doube quotes in the same line; if you can, lean towards one or the other for a full file.